casa/Arrays/ArrayLogical.h

```diff
@@ -390,13 +391,18 @@ template<class T> Bool anyOR (const T &val, const Array<T> &array);
 inline Bool allTrue (const Array<Bool>& array)
   { return allEQ (array, True); }
 
-// Is any all element true?
+// Is any element true?
 inline Bool anyTrue (const Array<Bool>& array)
   { return anyEQ (array, True); }
 
-// 
+// The same functions as above, but for selected axes.
+Array<Bool> partialAllTrue (const Array<Bool>& array,
+                            const IPosition& collapseAxes);
+Array<Bool> partialAnyTrue (const Array<Bool>& array,
+                            const IPosition& collapseAxes);
+
```

Comment:
wouldn't this make more sense as an overloaded variant of anyTrue?
partial does not sound axis reduction to me, more like slicing/windowing